### DISCUSSION TITLE
Port for Xcode

### DISCUSSION THREAD

**Discussion by rahat14** [daltonmenezes/aura-theme#122] [Port request]

**Comment by chris-c-thomas** [comment]
This would be a duplicate of this issue in regards to the JetBrains/Android Studio part: https://github.com/daltonmenezes/aura-theme/issues/9

Xcode should be a separate issue imo.

**Comment by daltonmenezes** [comment]
@rahat14 
@chris-c-thomas is right, the ideal is one port per issue to make the contributions better and easier to be done. That been said, I'm editing this issue to be only for Xcode, ok? 💜

**Comment by daltonmenezes** [comment]
@rahat14 @chris-c-thomas 

Done! 🎉 https://github.com/daltonmenezes/aura-theme/pull/235
